rate:     1000

# planner
rng_infl: 3
k_rep:    10

# outerloop parameters
ol_xi:    0.9
ol_wn:    2

# eta_reference parameters
er_kf_d:  3
er_kf_dd: 3

# innerloop parameters
il_v:     25
il_xi:    0.99
il_wn:    3

# wrench estimator
we_k0:    3